lib/opentuna-stack.ts

```diff
@@ -256,6 +257,23 @@ export class OpentunaStack extends cdk.Stack {
         priceClass: cloudfront.PriceClass.PRICE_CLASS_ALL,
       });
     }
+    // nested stack for log analysis
+    const OpentunaCloudFrontAccessLogsBucket = new s3.Bucket(this, "OpentunaCloudFrontAccessLogs");
```

Comment:
Use `OpenTUNALogs` as bucket name, we will put ALB logs to the same bucket later